content/blog/2021/07/camel-quarkus-release-2.1.0/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Camel Quarkus 2.1.0 Released"
+date: 2021-07-27
+authors: ["zbendiba"]
```

Comment:
I would have put the github  account here. so maybe **zbendhiba** ?

Reply:
yes, wrong username. Good catch !